app/(modals)/post/[id].tsx

```diff
@@ -1,15 +1,26 @@
 import { PageContainer } from "@/components/primitives/container";
 import { Header } from "@/components/primitives/header";
-import { Typography } from "@/components/primitives/typography";
-import { CommentInput } from "@app/(modals)/post/_features/comment-input";
 import { useComments } from "@app/(modals)/post/_features/comments.query";
-import type { CommentsData } from "@app/(modals)/post/_features/comments.schema";
+import { type CommentsData } from "@app/(modals)/post/_features/comments.schema";
 import { useOnePost } from "@app/(modals)/post/_features/one-post.query";
 import { Post } from "@app/(tabs)/posts/_features/post";
-import type { SinglePostData } from "@app/(tabs)/posts/_features/post.schema";
-import { FlashList, type ListRenderItem } from "@shopify/flash-list";
+import { FlashList } from "@shopify/flash-list";
 import { useLocalSearchParams } from "expo-router";
-import { Image, KeyboardAvoidingView, View } from "react-native";
+<<<<<<< HEAD
```

Comment:
be cautious, I've seen that on other files they other conflicts to fix

Reply:
is it due to the merging of main on this PR ?

Reply:
I don't really know, check the history change of the file

Reply:
fixed by https://github.com/info-telecom-strasbourg/InsidePSBS/pull/61/commits/81cb729f0a7a4706d5011f500ec831a1c446e91e